common/src/main/kotlin/serialization/DurationSerializers.kt

```diff
@@ -28,23 +29,64 @@ public sealed class DurationSerializer(private val unit: DurationUnit, name: Str
 }
 
 
+// nanoseconds
+
 /** Serializer that encodes and decodes [Duration]s in [whole nanoseconds][Duration.inWholeNanoseconds]. */
 public object DurationInWholeNanosecondsSerializer : DurationSerializer(NANOSECONDS, "DurationInWholeNanoseconds")
```

Comment:
Is there a particular reason for specifying that it's whole time units? 
It's serialized as a long which, to my knowledge, is always whole; right?

Reply:
I started implementing this using the `Duration.inWhole[Unit]s` properties, so I also included the "Whole" part in the names.
Leaving that part out would also be an option.

If we ever needed to introduce serializers that serialize `Duration`s to floating point values, the "Whole" part in the names would also help to distinguish between them. However, if we would find a meaningful name for the floating point ones, "Whole" might not be necessary.

Reply:
But to be honest, I would also prefer `DurationInSeconds` over `DurationInWholeSeconds`.

Reply:
I've decided to remove "Whole" from the names, let's worry about this in the future when we have to.